lib/main_containers.pm

```diff
@@ -404,5 +389,29 @@ sub load_container_tests {
         }
     }
     loadtest 'containers/bci_logs' if (get_var('BCI_TESTS'));
+
+    ## Helm chart tests. Add your individual helm chart tests here.
+    if (my $chart = get_var('HELM_CHART')) {
+        loadtest 'containers/k3s_ensure_essentials_installed';
+
+        my $spr_credentials_defined = !!(
+            get_var('SCC_REGISTRY', 0)
+            && get_var('SCC_PROXY_USERNAME', 0)
+            && get_var('SCC_PROXY_PASSWORD', 0)
+        );
+
+        loadtest 'containers/scc_login_to_registry' if is_sle() && $spr_credentials_defined;
+        set_var('K3S_ENABLE_COREDNS', 1);
+        if ($chart =~ m/rmt-helm$/) {
+            loadtest 'containers/charts/rmt';
+        } elsif ($chart =~ m/private-registry/) {
+            set_var('K3S_ENABLE_TRAEFIK', 1);
```

Comment:
I think the effect of this variable is to enable traefik in the k3s installation, which now happens before this variable is set.

So either this is not needed, or it needs to be set before `k3s_ensure_essentials_installed`